src/routes/docs/tutorials/pagination/step-5/+page.markdoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: tutorial
+title: Environment Setup
```

Comment:
Hey, this applies across the board, please make sure headings and titles are Sentence case instead of Title Case :D

Reply:
for sentence case, should titles like React and Appwrite be Capitialized?